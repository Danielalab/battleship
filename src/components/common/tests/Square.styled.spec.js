import 'jest-styled-components';
import { render } from '@testing-library/react';
import SquareStyled from '../Square.styled';

describe('Square.styled', () => {
  test('Should renders SquareStyled component', () => {
    const { container } = render(<SquareStyled backgroundColor="default" />);
    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
