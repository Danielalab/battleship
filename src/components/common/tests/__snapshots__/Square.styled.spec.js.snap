// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Square.styled Should renders SquareStyled component 1`] = `
.c0 {
  background: #2E4B84;
  border: 1px solid #1151B5;
  box-sizing: border-box;
  cursor: pointer;
  height: 40px;
  margin: 1px;
  width: 40px;
  -webkit-transition: background-color 0.05s ease-in;
  transition: background-color 0.05s ease-in;
}

.c0:hover > .target-icon {
  background-color: #007a00;
  visibility: visible;
}

<div>
  <div
    class="c0"
  />
</div>
`;
